api/controllers/files.js

```diff
@@ -55,7 +54,10 @@ const readdir = (path, stats, hidden, callback) => {
     }, err => {
       if (err) return callback(err);
 
-      entry = _.sortBy(entry, 'name');
+      entry.sort((a, b) => {
+        return a.name.localeCompare(b.name);
+      });
+
       callback(null, data);
```

Comment:
I don't know if the result of `String#localeCompare()` and `lodash.sortBy()` are the same

Reply:
Good call. I also not sure if it's supposed to return the whole entry or just entry.name. I'll do more testing to confirm.

On the other hand, [localCompare()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#Sorting_non-ASCII_characters) has the advantage of supporting non-ASCII characters.

Reply:
Just tried
```js
const _ = require('lodash')

let entry = [
{ name: 'check', age: 45 },
{ name: 'base', age: 68 },
{ name: 'éservé', age: 75 },
{ name: 'eserve', age: 23 }
]

//entry = _.sortBy(entry, 'name');

entry.sort((a, b) => {
  return a.name.localeCompare(b.name);
});

console.log(entry)
```

Both yield similar result.

Reply:
Thanks!